app/assets/stylesheets/application.css.scss

```diff
@@ -26,16 +26,22 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
 
 
 .post {
-  border-top: solid black 1px;
+  border-radius: 10px;
+  margin: 20px 10px;
+  padding: 5px 20px;
+  background-color: #fbfbfb;
   list-style: none;
-  h3 small span.offerer:not(:last-of-type):after {
-    content: ",";
-  }
-}
-.offers {
   h4 {
     font-weight: bold;
-    margin-top: 20px;
+    display: block;
+  }
+  .glyphicon-folder-open{
+    font-size: 140%;
+    margin: 0 5px;
+  }
+  .category{
+      margin-right: 10px;
```

Comment:
Line should be indented 4 spaces, but was indented 6 spaces<br>Properties should be ordered font-weight, margin-right